packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift

```diff
@@ -270,6 +289,7 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
             .mentions: true,
             .unsupportedBlockEditor: unsupportedBlockEnabled,
             .canEnableUnsupportedBlockEditor: unsupportedBlockCanBeActivated,
+            .mediaFilesCollectionBlock: true,
```

Comment:
is it possible to get this from a variable instead of setting it to `true`?  given the idea is that it should depend on a feature flag for now (we could also change this later given the features are not finished and connected on the host app yet, but just double checking here)

Reply:
🤕  - it's the demo app, forget about this